io.openems.edge.goodwe.et/src/io/openems/edge/goodwe/et/FeneconGoodweET.java

```diff
@@ -0,0 +1,1104 @@
+package io.openems.edge.goodwe.et;
```

Comment:
Why "FeneconGoodweET"? I know you copied it... but this is not a FENECON product!